packages/duplicate-package-checker-webpack-plugin/tests/jest/npm-2/index.test.js

```diff
@@ -1,5 +1,5 @@
-const webpack = require('webpack');
 const assert = require('assert');
```

Comment:
Question: Is it possible to reduce the size of this PR by reverting this change? Just from a surface level this doesn't seem to actually change anything.

Reply:
Our continuous integration build will fail if this change doesn't go through :(

Reply:
This is the build from the last commit showing that it failed at the linter step: https://github.com/cerner/terra-toolkit/actions/runs/5823395179/job/15790187109

Reply:
I see, in that case LGTM!